Backport new release notes structure to develop (#1316)
Avaiga/taipy-doc: docs/release-notes/4.2/index.md

---
title : Release Notes for version 4.2
---

> This is the list of changes to Taipy version 4.2.

Reviewer: ```suggestion
These are the updates and changes introduced in Taipy version 4.2.
```

  `InvalidSequence^` exception.<br/>
  See [issue #2322](https://github.com/Avaiga/taipy/issues/2322).
- :octicons-bug-24:{ .bug-icon title="Bug fix" } Taipy now only checks the compatibility
  of the installed *taipy-core* version with the versions set in the existing entities
  when the application runs either in experiment or in production mode.<br/>
  See [issue #2420](https://github.com/Avaiga/taipy/issues/2420).

# <strong><code>taipy-templates</code></strong>

## 4.2.0

<h4>New features</h4>

- :octicons-feed-plus-16:{ .plus-icon title="New feature" } A new application template
  named "classification" is available. For more information on creating a new Taipy
  application with the new "classification" template, refer to
  [Classification application template](../../tp_templates/applications/classification_app.md).
- :octicons-feed-plus-16:{ .plus-icon title="New feature" } New page templates are available
  for creating new pages on top of existing applications. For more information on creating
  new pages with the new page templates, refer to
  [Page templates](../../tp_templates/index.md#page-templates).

# <strong><code>taipy-enterprise</code></strong>

> ## 4.1.0

Author: ```suggestion
## 4.2.0
```
